rllib/utils/replay_buffers/tests/test_mixin_multi_agent_replay_buffer.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import unittest
+
+from ray.rllib.utils.replay_buffers.mixin_multi_agent_replay_buffer import \
+    MixInMultiAgentReplayBuffer
+from ray.rllib.policy.sample_batch import SampleBatch
+
+
+class TestMixInMultiAgentReplayBuffer(unittest.TestCase):
+    """Tests insertion and mixed sampling of the MixInMultiAgentReplayBuffer."""
+
+    capacity = 10
+
+    def _generate_data(self):
+        return SampleBatch(
+            {
+                "obs": [np.random.random((4,))],
+                "action": [np.random.choice([0, 1])],
+                "reward": [np.random.rand()],
+                "new_obs": [np.random.random((4,))],
+                "done": [np.random.choice([False, True])],
+            }
+        )
+
+    def test_mixin_sampling(self):
```

Comment:
a good practice is to write small unit tests that only test 1 thing / test.
at the very least, may be it's better to break this big test_mixin_sampling into test_mixin_sampling_mixed_batched
test_mixin_sampling_no_replay
test_mixin_sampling_replay_only
etc

Reply:
Ok I'll do the same for the other tests.